tutorials/admin/getting-started/join-network.md

```diff
@@ -1 +1,28 @@
-# join-network
+# Joining the Network
+
+Once you have [installed](installation.md) your Marble node on your server, it is time to connect it to other Marble
+nodes. 
+
+This will allow users of your node to discover data and services available on other nodes and will allow other users in
+the Marble community to make use of your node as well. By joining the network, you contribute to the Marble community
+and help make computational climate science more accessible and collaborative. Welcome!
+
+## Add your information to the registry
+
+To join the network, you need to add your node's information to the 
+[Marble Node Registry](https://github.com/DACCS-Climate/Marble-node-registry). This is a registry of all nodes in the
+network and is used by various tools and components of each Marble node to know how to pass information between nodes.
+
+To add your node's information, make a pull request to the 
```

Comment:
Marble Node Registry is already linked above, so no need to repeat it. If we want to include something here, a better option would be to do: `[pull request](https://github.com/DACCS-Climate/Marble-node-registry/pulls)`

Reply:
Also, I think we should change to "... to the Marble Node Registry with an updated `node_registry.json` file containing your node's information.